lib/controller.php

```diff
@@ -185,6 +185,14 @@ public function import_blob($blob) {
 			'_sha'         => $blob->sha,
 		) );
 
+		if(class_exists("WPCom_Markdown")) {
+			$wpcomMd = WPCom_Markdown::get_instance();
+			$args = $wpcomMd -> wp_insert_post_data($args, array(
+				"ID"           => $this->id,
```

Comment:
Thanks for catching this...
No. `wp_insert_post_data` just does the markdown conversion and puts everything in the right place (`post_content` vs. `post_content_filtered`